policy-controller/k8s/index/src/http_route.rs

```diff
@@ -22,16 +22,84 @@ pub enum InboundParentRef {
 #[derive(Clone, Debug, thiserror::Error)]
 pub enum InvalidParentRef {
     #[error("HTTPRoute resource does not reference a Server resource")]
-    DoesNotSelectServer,
+    DoesNotSelectServer(String),
 
     #[error("HTTPRoute resource may not reference a parent Server in an other namespace")]
-    ServerInAnotherNamespace,
+    ServerInAnotherNamespace(String),
 
     #[error("HTTPRoute resource may not reference a parent by port")]
-    SpecifiesPort,
+    SpecifiesPort(String),
 
     #[error("HTTPRoute resource may not reference a parent by section name")]
-    SpecifiesSection,
+    SpecifiesSection(String),
```

Comment:
what are the added `String`s here for? Are we going to add them to the error's formatted output?